libs/blocks/caas-marquee/caas-marquee.js

```diff
@@ -217,7 +239,7 @@ async function responseHandler(response, fnName) {
 
 async function getAllMarquees(promoId, origin) {
   const endPoint = isProd() ? API_CONFIGS.caas.prod : API_CONFIGS.caas.stage;
-  const payload = `originSelection=${origin}&promoId=${promoId}&language=en&country=US&perf=true`;
+  const payload = `originSelection=${origin}&promoId=${promoId}&language=en&country=US`;
```

Comment:
Why are we hard coding the language and locale?
we could use something like:
`const [language, country] = document.documentElement.lang.split('-');`

Reply:
This only gets the milo locales. We would need to then map those into the ones for CaaS. 

```
const [language, country] = document.documentElement.lang.split('-');
```

It was easier to just hardcode this for now since we are just supporting en/us rather than dealing with the mappings/imports (which bring potential latency issues for LH scores).